apps/axiom/src/components/AfterLogin/index.tsx

```
@@ -3,11 +1,82 @@
+import Link from 'next/link';
+import '../../theme/tailwind.scss';
 
-export const AfterLogin = () => {
+const OAuthButton = ({ href, iconSrc, bgColor, altText }) => {
+  return (
+    <Link href={href} style={{ flex: 1 }}>
+      <button style={{ ...buttonStyles, backgroundColor: bgColor }}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)